### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by nikhita**
xref kubernetes/community#1832

/assign brendandburns

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+Thanks for taking the time to join our community and start contributing!
+
+Please remember to read and observe the [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+This  project accepts contribution via github [pull requests](https://help.github.com/articles/about-pull-requests/).
```
Comment (SteveChurch) [inline CONTRIBUTING.md]: Double space found here `This  project`

Comment (SteveChurch) [inline CONTRIBUTING.md]: Double space found here `This project`

Comment (nikhita) [reply CONTRIBUTING.md]: @SteveChurch thanks for catching that!

fixed. :+1:

**Comment by brendandburns** [comment]
LGTM and thanks!

**Pull request merged**